src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -315,7 +315,20 @@ define(function (require, exports, module) {
                         selected={!layer.visible}
                         onClick={this._handleVisibilityToggle}>
                     </ToggleButton>
-                );
+                ),
+                iconClassModifier;
+
+            if (layer.isSmartObject()) {
```

Comment:
idle comment: It bugs me that `isSmartObject` is a prototype function and not a cached property.  Compare to `superSelectable` in models/layer